packages/build/tests/telemetry/tests.js

```diff
@@ -88,7 +83,10 @@ const runWithApiMock = async function (
 }
 
 test('Telemetry success generates no logs', async (t) => {
-  const { telemetryRequests } = await runWithApiMock(t, 'success', { telemetry: true })
+  const { telemetryRequests } = await runWithApiMock(t, 'success', {
+    telemetry: true,
+    featureFlags: 'buildbot_build_telemetry',
```

Comment:
Should `featureFlags` be added to `runApiWithApiMock()` since all tests but one passes it?

Reply:
I find them to be more explicit this way, but happy to do so 👍

Reply:
Addressed via https://github.com/netlify/build/pull/2438/commits/88419ae366d83d272229226fa481ce119eb444bf - notice that in order to test the default behaviour I'm setting the flag to `null` when calling the `runWithApiMock`